src/components/dashboard/main-content/course-enrollments/data/hooks.js

```diff
@@ -183,7 +183,6 @@ export const useCourseUpgradeData = ({
     }),
     enabled: !!applicableCouponCode && canUpgradeToVerifiedEnrollment,
   });
-
```

Comment:
nit: might keep this blank line for easier parsing of the logic, so the return statement isn't "grouped" with the above `useCourseRunMetadata` hook.